content/articles/2017-08-ggplot2-internship.Rmd

```diff
@@ -0,0 +1,87 @@
+---
+title: The summer of ggplot2
+slug: ggplot2-internship
+description: >
+  This summer I had the great fortune to accept the ggplot2 internship baton and
+  spend ten weeks developing new features and fixing bugs in ggplot2.
+author: Kara Woo
+date: 2017-08-25
+photo: 
+  url: https://unsplash.com/photos/DpF8nI-dQKA
+  author: Joe Shillington
+categories: [other]
+---
+
+This summer I had the great fortune to accept the
+[ggplot2 internship baton](https://twitter.com/dataandme/status/880952970602532864)
+from [Thomas Lin Pedersen](http://www.data-imaginist.com/2016/Becoming-the-intern/)
+and spend ten weeks developing new features and fixing bugs in ggplot2. My
+internship was a great experience, and I learned a ton from working with Hadley
+and digging into the ggplot2 codebase.
+
+## Daily life as a ggplot2 intern
+
+My ggplot2 work consisted of a few different things: issue and pull request
+triage, bug fixes, and developing new features. The tidyverse gets a lot of new
+issues opened on GitHub, so to help keep things organized I tagged each ggplot2
+issue with labels describing the type of issue and which part of ggplot2 it
+related to (scales, themes, layers, etc.). In some cases I helped the authors
+create reprexes so that we could diagnose the problems and determine when they
+were fixed. I triaged new pull requests as well, merging anything that was very
+minor, helping authors conform to the tidyverse style guide, and requesting
```

Comment:
"very minor" seems vaguely belittling to someone, I can't tell if it's you or the PRer or both. Maybe reword along these lines: "straightforward", "a clear win".